lib/dgram.js

```diff
@@ -264,15 +264,19 @@ function sliceBuffer(buffer, offset, length) {
 
 
 function fixBuffer(buffer) {
```

Comment:
`list` is a better name for the argument and, arguably, `fixBufferList` for the function.